portal-kernel/src/com/liferay/portal/kernel/servlet/PortletResponseHeadersHelperUtil.java

```diff
@@ -18,13 +18,21 @@
 
 import java.util.Map;
 
+import javax.servlet.RequestDispatcher;
 import javax.servlet.http.HttpServletResponse;
 
 /**
  * @author Tina Tian
  */
 public class PortletResponseHeadersHelperUtil {
 
+	public static RequestDispatcher getReloadHeadersRequestDispatcher(
+		RequestDispatcher requestDispatcher) {
+
+		return _getPortletResponseHeadersHelper().
+			getReloadHeadersRequestDispatcher(requestDispatcher);
+	}
+
 	public static void transferHeaders(
```

Comment:
@tinatian transferHeaders() is only used by getReloadHeadersRequestDispatcher(), right? Then it should not be an interface method. There should only be getReloadHeadersRequestDispatcher() in this interface, and the name can be refined if we do that. Please make this change and resend. Thanks!